ugvc/pipelines/single_cell_qc/single_cell_qc_pipeline.py

```diff
@@ -23,7 +23,7 @@
     OutputFiles,
     Thresholds,
 )
-from ugvc.utils.misc_utils import modify_jupyter_notebook_html
+from ugbio_core.report_utils import modify_jupyter_notebook_html
```

Comment:
modify_jupyter_notebook_html was moved to report_utils under ugbio_core:
https://github.com/Ultimagen/ugbio-utils/blob/a42a3a8b47ed292014f65750bbe046d44a7ba017/src/core/ugbio_core/report_utils.py#L1